enhancements/update/automatic-updates.md

```diff
@@ -0,0 +1,166 @@
+Before this enhancement she would need to construct a poller to run `oc adm upgrade --to-latest` or similar.
+With this enhancement, she can set `automaticUpdates` and does not need to write an external poller.
+
+### Risks and Mitigations
+
+Cluster-managed updates are a relatively new thing, and while we have dozens of OpenShift 4 releases out so far, we have had a few updates result in stuck clusters (like [this][machine-config-operator-drain-wedge].
+Allowing automatic updates would make it more likely that upgrade failures happen when there is no admistrator actively watching to notice and recover from the failure.
+This is mitigated by:
+
+* Alerting.
+    The cluster should fire alerts (FIXME: which ones?) when an upgrade gets into trouble.
+    Adminstrators can configure the cluster to push those alerts out to the on-call admistrator to recover the cluster.
+* Stability testing.
+    We are continually refining our CI suite and processing Telemetry from live clusters in order to assess the stability of each upgrade.
+    We will not place upgrades in production channels unless they have proven themselves stable in earlier testing, and we will remove upgrades from production channels if a live cluster trips over a corner case that we do not yet cover in pretesting.
+
+There are also potential future mitigations:
+
+* Phased releases, where Cincinnati spreads an update suggestion out over a configurable time window.
+    With hard cutovers and automatic updates, many clusters in a given channel could attempt a new update simultaneously.
+    If that update proves unstable, many of those upgrades would already be in progress by the time the first Telemetry comes back with failure messages.
+    A phased rollout would limit the number of simultaneously updating clusters to give Telemetry time to come back so we could stop recommending upgrades that proved unstable on live clusters not yet covered in pretesting.
+
+There is also a security risk where a compromised upstream Cincinnati could recommend cluster updates that were not in the cluster's best interest (e.g. 4.2.4 -> 4.1.0).
+This is mitigated by:
+
+* [The configurable `upstream` property][api-upstream].
+    Administrators who do not trust the default `upstream` to be sufficiently reliable may point their clusters at an upstream that they control which they can secure to their satisfaction.
+    The drawback to this approach would be that now they need tooling to approve graph updates as Red Hat makes changes, which will introduce some delays to security-fix rollouts.
+    But that approval would scale per-custom-Cincinnati instead of scaling per-cluster.
+
+There are also potential future mitigations:
+
+* The cluster-version operator could be taught to limit automatic upgrades to those which appear in the [signed metadata][cluster-version-operator-release-metadata] of either the source or target release.
```

Comment:
I don't think this is a mitigation that actually fit in the rest of design/release-engineering (or more likely, I didn't understand your suggestion).
In particular, it seems to conflict with the "rollouts based on Telemetry" point above.

Reply:
> I don't think this is a mitigation that actually fit in the rest of design/release-engineering...

This is a guard against malicious Cincinnati, whose risk exists independent of this enhancement.  This enhancement just increases exposure to the preexisting risk.